.travis.yml

```diff
@@ -61,9 +62,9 @@ matrix:
     - python: 3.5
       env: BUILD_DOCS=true
     - python: 3.5
-      env: USE_PYTEST=true PANDAS=pandas DELETE_FONT_CACHE=1 TEST_ARGS=
+      env: USE_PYTEST=true PANDAS=pandas DELETE_FONT_CACHE=1
     - python: 3.6
-      env: USE_PYTEST=true DELETE_FONT_CACHE=1 TEST_ARGS=
+      env: USE_PYTEST=true DELETE_FONT_CACHE=1 INSTALL_PEP8=pytest-pep8 RUN_PEP8=--pep8
```

Comment:
I think the one option handles optionally installing it on builds that use it, the other handles adding the argument to running pytest.